### ISSUE TITLE
[Feature] Use Filepond without its UI?

### ISSUE THREAD

**Issue by theJsTsGuy** [pqina/react-filepond#218]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

Hi File Pond Team,

Thank you very much for creating and painting this library!

I use this library in both the front-end (react) and back-end (laravel) for chunked file uploads. I now want to implement the UI by myself, which will be different from what File Pond provides, but with all features like uploading progress, file validation, etc.

I have searched the docs and couldn't find a way to handle all of the features File Pond provides through its instance, while disabling the UI,

### Describe the solution you'd like

Basically, I want a way to initialize File Pond without the UI, and use its instance methods and events to create my own UI.
Also, there is no event (apparently?), to get the progress of the file being uploaded to the server to listen for in chunk mode.

Please let me know if this is already I can do in FIle pond.

### Describe alternatives you've considered

I thought of implementing chunking and all of the validation by myself, but it is time-consuming and would require checking for a lot of edge cases.

**Comment by rikschennink** [comment]
Currently that's simply not possible, I'm slowly working on v5 which will have a separate ui and core so you can bring your own ui.

**Comment by theJsTsGuy** [comment]
Thank you @rikschennink! I appreciate the effort you are putting into this project!
Best Regards.
